pkg/daemon/controller/controller_test.go

```diff
@@ -654,9 +654,15 @@ func TestReconcile(t *testing.T) {
 
 			winSvcMgr := fake.NewTestMgr(test.existingServices)
 			c, err := NewServiceController(context.TODO(), "node", wmcoNamespace, Options{
-				Client:    clientfake.NewClientBuilder().WithObjects(clusterObjs...).Build(),
-				Mgr:       winSvcMgr,
-				cmdRunner: &fakePSCmdRunner{},
+				Client: clientfake.NewClientBuilder().WithObjects(clusterObjs...).Build(),
```

Comment:
Please test that the reboot-annotation is being applied to the node.

Reply:
How can i test this in the WICD unit tests? the fake client always returns the node obj as defined in the test, even if the reboot-annotation is "applied" using the fake client during reconcile

Reply:
are you sure it is being applied? I would think if the reconciler is being created by the same fake client it should be seen in the test, this can be tested by making a change in the unit test which uses the client to apply the annotation, and then checking immediately after if it is being applied

Reply:
turns out it was being applied I just had a bogus test... fixed now